overlord/devicestate/devicestate_test.go

```diff
@@ -1138,14 +1138,18 @@ func (s *deviceMgrSuite) TestDeviceManagerEmptyOperatingModeRun(c *C) {
 }
 
 func (s *deviceMgrSuite) TestStartOfOperationTimeFromSeedTime(c *C) {
-	st := s.state
+	st := state.New(nil)
+	// can't use device manager from the suite, we need to avoid o.StartUp()
+	mgr, err := devicestate.Manager(st, s.hookMgr, s.o.TaskRunner(), s.newStore)
```

Comment:
this looks a bit dirty, because those things taken from s will have a different state, maybe it's cleaner to have a separate suite? and have some control on Startup,  and/or have a helper in export_test.go to setup a  minimal DeviceManager ?

Reply:
Done (in a separate suite).